index.js

```diff
@@ -24,6 +24,14 @@ function clone(obj) {
   return copy;
 }
 
+function processOpts(opts_, outFile) {
+  var opts = clone(opts_);
+  if (outFile) opts.outFile = outFile;
+  if (opts.sourceMaps == 'inline')
+    opts.sourceMapContents = true;
+  return opts;
+}
```

Comment:
To be really nice to the engine, let's make this function define all the properties on `opts` that need to be used. This way it can be optimized by a single class without hitting deoptimizations later. It also acts nicely as a simple centralisation of info when I forget what all the options do.